app/views/handlers/list_action.py

```diff
@@ -15,9 +15,15 @@ def parent_location(self):
             block_id=self.rendered_block["parent_id"],
         )
 
+    @property
+    def section_routing_path(self):
```

Comment:
For context, as discussed, this should override `_get_routing_path` in `block.py` so that the routing path is only generated once.